core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/StaffScreenRobot.kt

```diff
@@ -16,6 +18,10 @@ class StaffScreenRobot @Inject constructor(
 ) {
 
     @Inject lateinit var robotTestRule: RobotTestRule
+
+    @Inject lateinit var sessionsApiClient: SessionsApiClient
```

Comment:
Sorry, It seems that  we should  use StaffApi for this 🙇

Reply:
@takahirom 
Sorry for the delay in contribution :bow:
I fixed it.
[:fix: change to staffApiClient](https://github.com/DroidKaigi/conference-app-2023/pull/1255/commits/f96388108352d6ed1daf3f72e69bc0e9863464fc)